source/config/configFlags.py

```diff
@@ -225,3 +225,49 @@ def _displayStringLabels(self):
 			# Translators: This is a label for the automatic update behaviour for add-ons.
 			self.DISABLED: _("Disabled"),
 		}
+
+
+@unique
+class OutputMode(DisplayStringIntEnum):
```

Comment:
Please make this more clearly a flag, this means the extra abstraction is less necessary

```suggestion
class OutputModeFlag(DisplayStringIntFlag):
```

Reply:
Sorry - just noted the issue with this raised in the docstring, can you let me know more about it

Reply:
The issue is that Python only considers non-zero powers of 2 to be canonical members of a `Flag` or `IntFlag`, so when iterating over the values of `OutputMode`, only `SPEECH` and `BRAILLE` would be returned. This can be worked around with `__members__`, but this would be trading one bit of nastiness for another. I tend to agree with you that it is a more pure approach though.